content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md

```diff
@@ -184,22 +196,23 @@ Given `_write_client_options=None`, the client uses synchronous mode when writin
 from influxdb_client_3 import InfluxDBClient3
 
 client = InfluxDBClient3(token="DATABASE_TOKEN",
-                         host="cluster-id.influxdb.io",
+                         host="{{< influxdb/host >}}",
                          org="",
                          database="DATABASE_NAME")
 ```
 {{% /code-placeholders %}}
 
 Replace the following:
 
-- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: an {{% cloud-name %}} [database token](/influxdb/cloud-dedicated/admin/tokens/) with read permissions on the databases you want to query
 - {{% code-placeholder-key %}}`DATABASE_NAME`{{% /code-placeholder-key %}}: the name of your {{% cloud-name %}} [database](/influxdb/cloud-dedicated/admin/databases/)
+- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: an {{% cloud-name %}} [database token](/influxdb/cloud-dedicated/admin/tokens/) with read permissions on the specified database
+
 
 ##### Initialize a client for batch writing
 
 The following example shows how to initialize a client for writing and querying the database.
-When writing data, the client uses batch mode with default options and
-invokes the callback function for the response.
+When writing data, the client will use batch mode with default options and will
+invoke the callback function for the response.
```

Comment:
Why the shift to future tense here? https://developers.google.com/style/tense

Reply:
Lol, I thought it was from an earlier suggestion you made on the initial PR.

Reply:
Haha, very well could've been. I find myself facepalming a lot over things like this 😄.